sdk/python/foundation-models/system/inference/text-to-image/safe-image-text-to-image-online-endpoint.ipynb

```diff
@@ -0,0 +1,761 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create an Azure AI Content Safety __(AACS)__ enabled Image to Image online endpoint (Preview)\n",
+    "### This notebook is under preview.\n",
+    "\n",
+    "### Steps to create an __AACS__ enabled __image-to-image__ online endpoint\n",
+    "1. Create an __AACS__ resource for moderating the request from user and response from the __image-to-image__ online endpoint.\n",
+    "2. Create a new __AACS__ enabled __image-to-image__ online endpoint with a custom [score_online.py](./aacs-scoring-files/score/score_online.py) which will integrate with the __AACS__ resource to moderate the response from the __image-text-to-image__ model and the request from the user, but to make the custom [score_online.py](./aacs-scoring-files/score/score_online.py) to successfully authenticated to the __AACS__ resource, we have 2 options:\n",
```

Comment:
Task name. Same comment as above.

The task should be image-to-image.
Have left the same comment on model asset PR - https://github.com/Azure/azureml-assets/pull/1785.